### DISCUSSION TITLE
ERROR auto train

### DISCUSSION THREAD

**Discussion by kitoshitachi** [huggingface/autotrain-advanced#672] [Q&A]
i dont know why it only show this, my data is json with two columns: words and ner_tags and its a list of strings 

INFO: 10.16.40.170:7879 - "GET /login/huggingface?__sign=eyJhbGciOiJFZERTQSJ9.eyJyZWFkIjp0cnVlLCJwZXJtaXNzaW9ucyI6eyJyZXBvLmNvbnRlbnQucmVhZCI6dHJ1ZX0sIm9uQmVoYWxmT2YiOnsia2luZCI6InVzZXIiLCJfaWQiOiI2NWFjYjYyYjA4NDRkOWUwZDY5N2VhYTgiLCJ1c2VyIjoiVmFtcGVlSHVudGVlIn0sImlhdCI6MTcxNzkxMTI4Nywic3ViIjoiL3NwYWNlcy9WYW1wZWVIdW50ZWUvdGVzdCIsImV4cCI6MTcxNzk5NzY4NywiaXNzIjoiaHR0cHM6Ly9odWdnaW5nZmFjZS5jbyJ9.O0V8Y7sxB4zO-R170XmJSxkK1qGa0RfsBrLRJj1BznEGk_U2Z7wcvpL9QkmhSwRYuX-fDrzz5v24iP6-OYRfAQ HTTP/1.1" 302 Found

ERROR | 2024-06-09 05:34:47 | autotrain.app.ui_routes:load_index:336 - Failed to get user and orgs: object of type '_TemplateResponse' has no len()

INFO: 10.16.14.230:13733 - "GET /?__sign=eyJhbGciOiJFZERTQSJ9.eyJyZWFkIjp0cnVlLCJwZXJtaXNzaW9ucyI6eyJyZXBvLmNvbnRlbnQucmVhZCI6dHJ1ZX0sIm9uQmVoYWxmT2YiOnsia2luZCI6InVzZXIiLCJfaWQiOiI2NWFjYjYyYjA4NDRkOWUwZDY5N2VhYTgiLCJ1c2VyIjoiVmFtcGVlSHVudGVlIn0sImlhdCI6MTcxNzkxMTI4Nywic3ViIjoiL3NwYWNlcy9WYW1wZWVIdW50ZWUvdGVzdCIsImV4cCI6MTcxNzk5NzY4NywiaXNzIjoiaHR0cHM6Ly9odWdnaW5nZmFjZS5jbyJ9.O0V8Y7sxB4zO-R170XmJSxkK1qGa0RfsBrLRJj1BznEGk_U2Z7wcvpL9QkmhSwRYuX-fDrzz5v24iP6-OYRfAQ HTTP/1.1" 307 Temporary Redirect

INFO: 10.16.40.170:61339 - "GET /?__sign=eyJhbGciOiJFZERTQSJ9.eyJyZWFkIjp0cnVlLCJwZXJtaXNzaW9ucyI6eyJyZXBvLmNvbnRlbnQucmVhZCI6dHJ1ZX0sIm9uQmVoYWxmT2YiOnsia2luZCI6InVzZXIiLCJfaWQiOiI2NWFjYjYyYjA4NDRkOWUwZDY5N2VhYTgiLCJ1c2VyIjoiVmFtcGVlSHVudGVlIn0sImlhdCI6MTcxNzkxMTI4Nywic3ViIjoiL3NwYWNlcy9WYW1wZWVIdW50ZWUvdGVzdCIsImV4cCI6MTcxNzk5NzY4NywiaXNzIjoiaHR0cHM6Ly9odWdnaW5nZmFjZS5jbyJ9.O0V8Y7sxB4zO-R170XmJSxkK1qGa0RfsBrLRJj1BznEGk_U2Z7wcvpL9QkmhSwRYuX-fDrzz5v24iP6-OYRfAQ HTTP/1.1" 307 Temporary Redirect

INFO: Uvicorn running on http://0.0.0.0:7860 (Press CTRL+C to quit)

**Comment by KryslynnB** [comment]
Suggest An Answer

**Reply by abhishekkrthakur** [reply]
seems the app has started fine and i dont see any errors. need more information before providing answer :)
